packages/web-react/src/components/Grid/GridItem.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ElementType } from 'react';
+import classNames from 'classnames';
+import { useStyleProps } from '../../hooks';
+import { SpiritGridItemProps } from '../../types';
+import { useGridItemStyleProps } from './useGridItemStyleProps';
+
+const GridItem = <T extends ElementType = 'div'>(props: SpiritGridItemProps<T>): JSX.Element => {
+  const { elementType: ElementTag = 'div', children, ...restProps } = props;
+  const { classProps, styleProps: gridItemStyle, props: modifiedProps } = useGridItemStyleProps(restProps);
+  const { styleProps, props: otherProps } = useStyleProps(modifiedProps);
+
+  styleProps.style = {
```

Comment:
I do not like this style of extending the values of the object. E. g. let's treat that object like it is immutable and rather create a new object from the merge of them both.

```js
const gridItemStyleProps = { style: {
  ...styleProps.style,
  ...gridItemStyle,
}}
```

I think that it is safe in this case but it can be potentially problematic in the future (some reference errors). Thanks.